core/templates/services/translation-file-hash-loader.service.ts

```diff
@@ -32,10 +33,10 @@ angular.module('oppia').factory('TranslationFileHashLoaderService', [
         options.key,
         options.suffix
       ].join('');
-      return $http.get(
+      return TranslationsBackendApiService.loadTranslationFileHash(
```

Comment:
Here is what I found from documentation:

>... $translateProvider know of a translation table. What does that mean? It's actually quite simple. angular-translate expects translation tables as JSON objects. So all you have to do, is to represent your translation data as JavaScript hash object, or, when loading remotely, as a JSON object....

from here: [https://angular-translate.github.io/docs/#/guide/02_getting-started]

It could be different name but I found it appropriate to documentation.